src/types.ts

```diff
@@ -315,9 +315,16 @@ export interface RootProps {
   announce: Announce;
 }
 
+export interface BodyWrapperProps
+  extends Omit<RootProps, 'changeSortOrder' | 'rect' | 'pageInfo' | 'setPageInfo' | 'footerContainer' | 'translator'> {
```

Comment:
Not sure about this.  Almost half the props on RootProps are being omitted and three are added. As I mentioned in here https://github.com/qlik-oss/sn-table/pull/663 I'm not sure we should omit these as they are actually passed.

Reply:
First I added them to RootProps but Li suggested the extended ones. The only benefit of omitting the unnecessary ones is we don't need to add every single RootProps in TableBodyWraper test component.

Reply:
ok go for this now and we might refactor it later